### PR TITLE
ui: fix the color of the logout link

### PR DESCRIPTION
**Pull request by couchand**
Release note: None

---

Before:
screen shot 2018-10-16 at 3 02 06 pm

After:
screen shot 2018-10-16 at 3 00 42 pm

### CODE REVIEW

File: pkg/ui/styl/layout/navigation-bar.styl [hunk 1]
```diff
@@ -97,6 +97,7 @@ $subnav-underline-height = 3px
         display inline
         padding 0
         text-decoration none
+        color inherit
```
Comment (vilterp) [inline pkg/ui/styl/layout/navigation-bar.styl]: Hm, I thought it would inherit or "cascade" by default…

**Review by vilterp — APPROVED** [review]
Good catch.

File: pkg/ui/styl/layout/navigation-bar.styl (hunk 1 above)
Comment (couchand) [reply pkg/ui/styl/layout/navigation-bar.styl]: css 🤦‍♂️

**Pull request merged**